os/tools/compression/src/compress.c

```diff
@@ -244,6 +250,9 @@ static void compress_file(int block_size, int type, char *in_file, char *out_fil
 	if (out_fd > 0) {
 		close(out_fd);
 	}
+	if (compression_fail) {
```

Comment:
Is it okay not to delete the output file for compression?

Reply:
the output file for compression is opened with O_TRUNC flag.
https://github.com/Samsung/TizenRT/blob/114810f9e6cf82f01086e0092c3cb961ff3dc1fc/os/tools/compression/src/compress.c#L80
So I think we need not delete the output file